Source/Tests/LinuxSyscalls/Syscalls.cpp

```diff
@@ -226,89 +211,156 @@ static bool IsSupportedByInterpreter(std::string const &Filename) {
   return false;
 }
 
-uint64_t ExecveHandler(const char *pathname, char* const* argv, char* const* envp, ExecveAtArgs *Args) {
+static bool IsShebangFilename(std::string const &Filename) {
```

Comment:
If we have to implement this twice anyway, why not use POSIX `open` and make `IsShebangFilename` call `IsShebangFD`?

Reply:
Good call. Resolved.